src/generic_ui/scripts/ui.spec.ts

```diff
@@ -76,7 +76,6 @@ describe('UI.UserInterface', () => {
         userId: userId,
         name: userName,
         imageData: 'testImageData',
```

Comment:
I agree with that -- only reason is, I guess, crappy JSON parsers...which doesn't apply here.